src/python/pants/backend/jvm/tasks/coverage/cobertura.py

```diff
@@ -55,8 +55,7 @@ def cobertura_jar(**kwargs):
     register_jvm_tool(register,
```

Comment:
Please amend the comment above, maybe ", but not any of its dependencies except an slf4j implementation for logging. We leave this out to trade ..." and reference a new issue that tracks shading cobertura as a more complete solution.'